csharp/IceStorm/Weather/Station/Program.cs

```diff
@@ -3,7 +3,8 @@
 using ClearSky;
 using System.Diagnostics;
 
-// Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program exits.
+// Create an Ice communicator. We'll use this communicator to create proxies, manage  outgoing connections, and create
```

Comment:
Extra space snuck in there.
```suggestion
// Create an Ice communicator. We'll use this communicator to create proxies, manage outgoing connections, and create
```